backend/src/binaryInstaller.ts

```diff
@@ -40,33 +41,28 @@ class BinaryInstaller {
 
   async installBinary(installer: Installer) {
     return new Promise(async (resolve, reject) => {
-      var tmpDir = tmp.dirSync({ unsafeCleanup: true, keep: true })
-
       // Stop and remove old binaries
-      await this.migrateBinaries(installer.binaryPath())
-
-      // Download and install binaries
-      await this.download(installer, tmpDir)
+      // await this.migrateBinaries(installer.binaryPathCLI())
```

Comment:
We will want to put in some cleanup commands here to remove old binaries that are located in /Program Files/remoteit-bin/ or in /usr/local/bin or /usr/bin etc... that's what this line was doing, so we'll want to bring it back and update it.